Marlin/src/inc/SanityCheck.h

```diff
@@ -514,10 +514,8 @@ static_assert(X_MAX_LENGTH >= X_BED_SIZE && Y_MAX_LENGTH >= Y_BED_SIZE,
 /**
  * LCD Lightweight Screen Style
  */
-#if DISABLED(U8GLIB_ST7920)
-  #if ENABLED(LIGHTWEIGHT_UI)
-      #error "LIGHTWEIGHT_UI requires U8GLIB_ST7920-based display."
-  #endif
+#if ENABLED(LIGHTWEIGHT_UI) && DISABLED(U8GLIB_ST7920)
```

Comment:
This will never evaluate true since LIGHTWEIGHT_UI enablement is wrapped in an if-enabled so no user warning (so identical behavior as prior to PR)